### DISCUSSION TITLE
Which mythical creature traditionally lives under a bridge?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#738] [Galaxy Trivia] [answered]
**Bonus:** In which US city can you find one?

**Comment by peterbaccaro** [comment, marked as answer]
The mythical creature that traditionally lives under a bridge is a **troll**. 
You can find one in the Fremont neighborhood of **Seattle**.  The large concrete sculpture of a troll was created by a group of local artists in 1990. The troll is a popular tourist attraction and has become an iconic symbol of the city. It is depicted as emerging from under the Aurora Bridge and clutching an actual Volkswagen Beetle in its hand, which has become a distinctive feature of the sculpture.

**Comment by thuongtruong1009** [comment]
The mythical creature that traditionally lives under a bridge is a troll. In the United States, the city of Seattle, Washington is known for having a famous troll sculpture called the "Fremont Troll." It is located under the Aurora Bridge in the Fremont neighborhood of Seattle. The Fremont Troll is a popular tourist attraction and a representation of the mythical troll creature.

**Comment by SANKARAMDAS** [comment]
Trolls in modern tales for children often live under bridges, menacing travelers and exacting tasks or tolls.
Troll, in early Scandinavian folklore, giant-mythology, monstrous being, sometimes possessing magic powers.
In the plays of the Norwegian dramatist Henrik Ibsen, especially Peer Gynt (1867) and The Master Builder (1892), trolls are used as symbols of destructive instincts. Trolls in modern tales for children often live under bridges, menacing travelers and exacting tasks or tolls.

**Comment by techvmuppa** [comment]
The mythical creature that traditionally lives under a bridge is a troll. In Scandinavian folklore, trolls are supernatural beings that are often depicted as living in mountains, caves, or forests. They are also known for being solitary creatures, and sometimes are described as having a great fondness for riddles. In many stories, trolls are known for causing trouble for humans who cross their path, such as demanding payment to cross a bridge that they guard. This has led to the phrase "trolling for tolls" to describe this type of behavior.
